README.md

```diff
@@ -45,7 +45,7 @@ See `KeychainExample` for fully working project example.
 
 Both `setGenericPassword` and `setInternetCredentials` are limited to strings only, so if you need to store objects etc, please use `JSON.stringify`/`JSON.parse` when you store/access it.
 
-### `setGenericPassword(username, password, [{ accessControl, accessible, accessGroup, service }])`
+### `setGenericPassword(username, password, securityLevel, [{ accessControl, accessible, accessGroup, service }])`
```

Comment:
the way `securityLevel` was added as a param seems like a breaking change.(?)
would it be possible to add this into the params object? (`{ accessControl, accessible, accessGroup, service }`). 

this is just a quick observation, I'll try to do proper review some time next week

Reply:
Thanks! It's optional, I think I did the docs wrong... I will wait for more feedback and will fix everything at once then.

Reply:
Is there any progress on this merge? Very nice feature.